src/main/resources/assets/admin/common/js/form/inputtype/radiobutton/RadioButton.ts

```diff
@@ -99,25 +98,25 @@ export class RadioButton
     }
 
     private readConfig(inputConfig: { [element: string]: { [name: string]: string }[]; }): void {
-        let options: { label: string; value: string; }[] = [];
+        const options: { label: string; value: string; }[] = [];
```

Comment:
We should declare this as a type as it's used throughout the class.

Reply:
Updated